src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/ViolationTicketMapperTest.java

```diff
@@ -15,7 +15,7 @@
 public class ViolationTicketMapperTest extends BaseTestSuite{
 
 	private ViolationTicketMapper violationTicketMapper;
-
+	
```

Comment:
If you update your IDE to always trim trailing whitespace on save, we won't be flipping back and forth with these whitespace changes.

Reply:
My Eclipse IDE has this setting -- unknown why this one slipped through